include/nitro/broken_options/option/group.hpp

```diff
@@ -41,34 +44,190 @@ namespace broken_options
     class group
     {
     public:
-        group(const std::string& name, const std::string& description = std::string(""))
-        : name_(name), description_(description)
+        std::string name;
+        std::string description;
+
+    public:
+        group(std::set<std::string>& all_argument_names, const int& position,
```

Comment:
Don't pass trivial types by const reference.
```suggestion
        group(std::set<std::string>& all_argument_names, int position,
```